content/docs/latest/languagemanual-sortby_27362045.md

```diff
@@ -190,7 +190,16 @@ SELECT col1, col2 FROM t1 DISTRIBUTE BY col1 SORT BY col1 ASC, col2 DESC
 
 ```
 
- 
+Note that columns are specified by name, not by position number. However in [Hive-28572](https://issues.apache.org/jira/browse/HIVE-28572) and later, columns can be specified by position when configured as follows:
 
- 
+* set [hive.orderby.position.alias](https://hive.apache.org/docs/latest/configuration-properties_27842758/#hiveorderbypositionalias)=true; (True is the default)
+* set [hive.cbo.enable](https://hive.apache.org/docs/latest/configuration-properties_27842758/#hivecboenable)=true; (True is the default)
```

Comment:
nit: mention to the defaults could be removed

Reply:
Removed